src/test/java/training/sign_in/responsesuccessfully.json

```diff
@@ -0,0 +1,10 @@
+{
+  "user": {
+    "id": "#number",
+    "email": "#string",
+    "username": "#string",
+    "bio": "#null",
```

Comment:
is this always null? if not, add the data type with double "#" in order to indicate this field is optional